lib/lz4.h

```diff
@@ -346,6 +346,8 @@ LZ4LIB_API int LZ4_loadDict (LZ4_stream_t* streamPtr, const char* dictionary, in
  */
 LZ4LIB_API int LZ4_compress_fast_continue (LZ4_stream_t* streamPtr, const char* src, char* dst, int srcSize, int dstCapacity, int acceleration);
 
+LZ4LIB_API int LZ4_DictSize (LZ4_stream_t* LZ4_dict, int dictSize);
```

Comment:
A few conventions : 
- function names start in lowercase, excluding the prefix
- new functions shall be documented. What does it do ? Set a new dictSize ? Get a current dictSize ? What are the limitations ? What is the parameter for ? What happens in case of error ?
- Generally, function name starts with a verb/action, to better qualify the effect, for example `LZ4_setDictSize()` or `LZ4_reduceDictSize()`.
- New symbols do not start their life directly in "stable" area. They have to spend some time in "staging" area below, to prove their worth and collect user feedback. As a consequence, the qualifier changes to `LZ4LIB_STATIC_API`.

Reply:
I think I solved your comments. Also added a fuzzing test to make sure the changes are working properly

Reply:
> Also added a fuzzing test to make sure the changes are working properly

Great !